src/utils/valideringUtils.ts

```diff
@@ -54,6 +55,13 @@ export const validerArbeidsgiver = (orgNummer?: string): string | undefined => {
   return undefined;
 };
 
+export const validerBehandlerRef = (behandlerRef?: string) => {
```

Comment:
navnet validerBehandlerRef for meg leser jeg som at den skal validere behandlerref, men her sjekker man vel egentlig etter feil? Kanskje man kan vri navnet til noe litt mer i den retninga?